Source/WebCore/loader/HistoryController.cpp

```diff
@@ -381,7 +381,10 @@ void HistoryController::goToItemForNavigationAPI(HistoryItem& targetItem, FrameL
 
         protectedThis->m_policyItem = nullptr;
 
-        if (!result)
+        // For Navigation API navigations covered by HistoryController:goToItemForNavigationAPI, WebContent processes sometimes
+        // know about an item the UI process doesn't know about. In those cases, policy checks will happen elsewhere, and the
+        // traversal should occur
+        if (result == ShouldGoToHistoryItem::No)
```

Comment:
Not consistent check. Checking with Yes or No should be consistent with original check.

Reply:
Oh, actually this is tai-state. NM